lib/team_api/joiner.rb

```diff
@@ -121,21 +142,17 @@ def team_members
     # - Strings that are GitHub usernames
     # - Hashes that contain an 'email' property
     # - Hashes that contain a 'github' property
-    def join_team_list(team_list)
+    def join_team_list(team_list, errors)
       (team_list || []).map! do |reference|
-        member = team_member_from_reference reference
+        member = @team_indexer.team_member_from_reference reference
```

Comment:
You should actually just use `team_indexer` instead of `@team_indexer` here and everywhere else outside of the constructor.